app/plugins/ee-edge-caching.md

```diff
@@ -0,0 +1,34 @@
+---
+id: page-plugin
+title: Plugins - EE Edge Caching
+header_title: EE Edge Caching
+header_icon: /assets/images/icons/plugins/rate-limiting.png
+breadcrumbs:
+  Plugins: /plugins
+nav:
+  - label: Getting Started
+    items:
+      - label: Requesting Access
+---
+
+Lorem ipsum dolor sit amet, consectetur adipisicing elit. Minus aliquid ea assumenda ipsa iure officia obcaecati quod, cupiditate exercitationem mollitia animi veritatis eum deleniti, et libero quos cumque consequuntur dolore.
```

Comment:
The Edge Caching plugin for Kong Enterprise Edition makes it fast and easy to configure caching of responses and serving of those cached responses to matching requests.